Aardvark/ARKEmailBugReporter.h

```diff
@@ -53,6 +54,20 @@
 @end
 
 
+typedef void (^ARKEmailBugReporterCustomPromptCompletionBlock)(ARKEmailBugReportConfiguration *_Nullable configuration);
+
+@protocol ARKEmailBugReporterPromptingDelegate <NSObject>
+
+@required
+
+/// Called on the main thread when a bug is filed to signal that a bug report prompt should be presented to the user.
+/// The `completion` should be called with either an updated configuration to present the email dialog, or `nil` to signal that the prompt was cancelled.
```

Comment:
on what thread should `completion` be called?

Reply:
main 👍